core/templates/dev/head/services/context.service.spec.ts

```diff
@@ -50,6 +50,11 @@ describe('Context service', () => {
     it('should correctly retrieve the correct entity type', () => {
       expect(ecs.getEntityType()).toBe('exploration');
     });
+
+    it('should correctly check that page allows editing of RTE components',
```

Comment:
For the "false" cases, this should say something like: it should affirm that the page forbids editing of RTE components.

Reply:
done